dispatcher/MongoConnect.py

```diff
@@ -214,7 +206,7 @@ def aggregate_status(self):
                     if dt > self.timeout:
                         self.log.debug(f'{doc["host"]} reported {int(dt)} sec ago')
                         status = DAQ_STATUS.TIMEOUT
-                        if self.host_config[doc['host']] == 'tpc':
+                        if hc[doc['host']] == 'tpc':
```

Comment:
What does this if statement mean in linked mode? That means also readers4 and 5 which can be problematic

Reply:
same in line 235

Reply:
`hc` is the host config which refers to the physical grouping not the logical grouping.